README.md

```diff
@@ -1,2 +1,199 @@
+  </ul>
+</details>
+
+<details>
+<summary>React</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The following are the key features of the application.
+
+- [ ]  Linters setup
+- [ ]  React setup
+- [ ]  Initialize Bookstore app with components (React only)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Click the live demo to access the application.
+- coming soon ...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+- Code editor: VS code or other
+- Web browser: like Google Chrome
+- git-scm
+```sh
+  npm install
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+  cd my-folder
+  git clone git@github.com:mulugithub/bookstore.git
+```
+
+### Install
+
+To run the project, open the project using a live server from VS code editor.
+```sh
+  cd math-magician
+  code .
+  npm install
+```
+
+### Usage
+To run this project:
+```sh
+  npm  start
+```
+### Run tests
+To run tests, run the following command:
+To check for html errors run:
+```sh
+  npx hint .
+```
+To check for css errors run:
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+To check for js errors run:
+```sh
+  npx eslint .
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> This project is authored by.
+
+👤 **Author1**
+
+- GitHub: [@mulugithub](https://github.com/mulugithub)
+- LinkedIn: [@muluken-zemed](https://www.linkedin.com/in/muluken-zemed-2b6a38167/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> The following features will add to the project.
+- [ ]  Deploy the App
+- [ ]  Add reducers and actions.
+- [ ]  Use Redux in React components.
+- [ ]  Connect to API.
+- [ ]  Styling.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it a star on GitHub.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank Microverse team to give this project and the MV code reviewers.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [optional] Great job. Please kindly fix your MIT license for it is broken. we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
![Screenshot_8](https://github.com/mulugithub/bookstore/assets/65298048/3e02e73f-4368-4431-a7ce-b1bce08eb2fe)